modules/4_nodes/nodes.tf

```diff
@@ -23,18 +23,46 @@ resource "random_id" "label" {
     byte_length = "2"
 }
 
+data "ignition_systemd_unit" "restart-NetworkManager" {
+    name        = "NetworkManager.service"
+    dropin {
+        name    = "restart.conf"
+        content = <<EOF
+[Service]
+Restart=always
+RuntimeMaxSec=300
+EOF
+    }
+}
```

Comment:
Please add a comment that this is super hacky and will be removed the moment we have a proper fix

Reply:
Will do.